auth0/src/main/java/com/auth0/android/provider/OAuthManager.java

```diff
@@ -98,6 +98,16 @@ void startAuthentication(Activity activity, String redirectUri, int requestCode)
     @SuppressWarnings("ConstantConditions")
     @Override
     boolean resume(AuthorizeResult result) {
+        // Should never happen where the algorithms required are not present, but just in case it does,
```

Comment:
I'd move this check to the WebAuthProvider Builder constructor, or at least the `start` method. Failing faster than having to wait to come back from the browser sounds better IMO.